arithmetization/src/main/java/net/consensys/linea/zktracer/container/stacked/CountOnlyOperation.java

```diff
@@ -45,4 +44,9 @@ public void add(final int operationCount) {
   public int lineCount() {
```

Comment:
Could we add a `Preconditions.CheckArgument(thisTransactionCount + operationCount >= 0)` here ?

Reply:
added